nextclade/config/auspice_config.json

```diff
@@ -0,0 +1,50 @@
+{
+  "title": "Genomic epidemiology of Influenza",
+  "build_url": "https://github.com/neherlab/nextclade_data_workflows",
```

Comment:
Should we update this build URL to reference this repo instead?

Reply:
yes, updated!